.github/workflows/tag.yml

```diff
@@ -14,23 +14,14 @@ jobs:
         tennens: [default, brb, demo, int, n21, open, thr ]
     steps:
       - uses: actions/checkout@v2
```

Comment:
why changing the workflows, how the relate to dependency updates?

Reply:
ignore it for now, this is not finished yet.

Reply:
when this is not finished --> it should not be merged. 
when this is not part of the ticket --> it should be a separate pr.